README.md

```diff
@@ -28,4 +29,44 @@ The data set was made from the several PCAP files in the [2018](https://archive.
 # zeek -r wrccdc.2018-03-24.101533000000000.pcap -r wrccdc.2018-03-24.101551000000000.pcap -r wrccdc.2018-03-24.101610000000000.pcap -r wrccdc.2018-03-24.101629000000000.pcap -r wrccdc.2018-03-24.101737000000000.pcap -r wrccdc.2018-03-24.101939000000000.pcap -r wrccdc.2018-03-24.102051000000000.pcap -r wrccdc.2018-03-24.102126000000000.pcap -r wrccdc.2018-03-24.102233000000000.pcap -r wrccdc.2018-03-24.102443000000000.pcap -r wrccdc.2018-03-24.102602000000000.pcap -r wrccdc.2018-03-24.102643000000000.pcap -r wrccdc.2018-03-24.102717000000000.pcap -r wrccdc.2018-03-24.102733000000000.pcap -r wrccdc.2018-03-24.102747000000000.pcap -r wrccdc.2018-03-24.102831000000000.pcap -r wrccdc.2018-03-24.102920000000000.pcap -r wrccdc.2018-03-24.103009000000000.pcap -r wrccdc.2018-03-24.103049000000000.pcap -r wrccdc.2018-03-24.103117000000000.pcap -r wrccdc.2018-03-24.103152000000000.pcap -r wrccdc.2018-03-24.103210000000000.pcap -r wrccdc.2018-03-24.103224000000000.pcap -r wrccdc.2018-03-24.103256000000000.pcap -r wrccdc.2018-03-24.103420000000000.pcap -r wrccdc.2018-03-24.103630000000000.pcap local
 ```
 
-This produced the logs in Zeek default and NDJSON formats. As ZSON is not yet output directly by Zeek, the ZSON-format logs were created by sending each Zeek default log through `zq` (e.g., `zq conn.log > conn.ndjson`).
+This produced the logs in Zeek default and NDJSON formats. As ZNG/BZNG are not yet output directly by Zeek, these logs were created by sending each Zeek default log through `zq`, e.g.:
+
+```
+# for file in zeek-default/*; do gzcat $file | zq -f zson - | gzip > zson/$(basename $file | sed 's/\.log\.gz//').zson.gz ; done
+# for file in zeek-default/*; do gzcat $file | zq -f bzson - | gzip > bzson/$(basename $file | sed 's/\.log\.gz//').bzson.gz ; done
```

Comment:
Nit1. Use backslashes and other constructions to trim line length. At https://github.com/mccanne/zq-sample-data/tree/zng-redo#creation you can see this needing scroll bars.
Nit2: shellcheck your example bash ;)
```
mkdir -p zson && \
for file in zeek-default/*
do
  gzcat "$file" \
      | zq -f zson - \
      | gzip > zson/"$(basename "$file" | sed 's/\.log\.gz//')".zson.gz 
done
```

Reply:
RE: Nit1. Hunh, ok... fair point. I guess I personally don't find horizontal scrolling bothersome, particularly when it's reference material that I expect most people to skip over anyway. But now that you say it, I realize I'm probably the oddball in this regard. I've switched to your proposal.

RE: Nit2. Duh. Thanks. I was so focused on shellcheck passing on the main script that I never thought to do it on my README examples. :)